app/models/content_store_payload_version.rb

```diff
@@ -0,0 +1,41 @@
+class ContentStorePayloadVersion < ActiveRecord::Base
+  def self.increment(content_item_id:)
```

Comment:
I had it as a regular argument at first but it didn't read all that well when it is called.
`ContentStorePayloadVersion.increment(content_item_id: 10)` looked better to me than `ContentStorePayloadVersion.increment(10)`.